src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -436,42 +436,37 @@ public function getEditorAjax($filePath, $line)
      * act as an Ajax request. The editor must be a
      * valid callable function/closure
      *
-     * @throws UnexpectedValueException  If editor resolver does not return a boolean
-     * @param  string                   $filePath
-     * @param  int                      $line
-     * @return mixed
+     * @param  string $filePath
+     * @param  int    $line
+     * @return array
      */
     protected function getEditor($filePath, $line)
     {
-        if ($this->editor === null && !is_string($this->editor) && !is_callable($this->editor))
-        {
-            return false;
+        if (!$this->editor || (!is_string($this->editor) && !is_callable($this->editor))) {
+            return [];
         }
-        else if(is_string($this->editor) && isset($this->editors[$this->editor]) && !is_callable($this->editors[$this->editor]))
-        {
+
+        if (is_string($this->editor) && isset($this->editors[$this->editor]) && !is_callable($this->editors[$this->editor])) {
            return [
                 'ajax' => false,
-                'url' => $this->editors[$this->editor],
+                'url'  => $this->editors[$this->editor],
             ];
         }
-        else if(is_callable($this->editor) || (isset($this->editors[$this->editor]) && is_callable($this->editors[$this->editor])))
-        {
-            if(is_callable($this->editor))
-            {
+
+        if (is_callable($this->editor) || (isset($this->editors[$this->editor]) && is_callable($this->editors[$this->editor]))) {
+            if (is_callable($this->editor)) {
                 $callback = call_user_func($this->editor, $filePath, $line);
-            }
-            else
-            {
+            } else {
                 $callback = call_user_func($this->editors[$this->editor], $filePath, $line);
             }
 
             return [
                 'ajax' => isset($callback['ajax']) ? $callback['ajax'] : false,
-                'url' => (is_array($callback) ? $callback['url'] : $callback),
+                'url'  => isset($callback['url']) ? $callback['url'] : $callback,
```

Comment:
`isset($callback['url'])` apparently works and correctly returns `false` if `$callback` is a string. But it's real unexpected for me to see `isset` called on a string.

Would you please clarify this by adding the following statement on line 462:

```php
if (is_string($callback)) {
    $callback = [
        'ajax' => false,
        'url' => $callback;
    ];
}
```